domain/robstoll-lib/atrium-domain-robstoll-lib-jvm/src/main/kotlin/ch/tutteli/atrium/domain/robstoll/lib/creating/pathAssertions.kt

```diff
@@ -20,4 +20,9 @@ fun <T : Path> _existsNot(assertionContainer: Expect<T>): Assertion =
     ExpectImpl.builder.createDescriptive(assertionContainer, NOT_TO, RawString.create(EXIST)) { it.notExists }
 
 fun <T : Path> _parent(assertionContainer: Expect<T>): ExtractedFeaturePostStep<T, Path> =
-    ExpectImpl.feature.manualFeature(assertionContainer, PARENT) { parent }
+    ExpectImpl.feature.extractor(assertionContainer)
+        .methodCall("parent")
```

Comment:
That's a bit misleading as in kotlin it is not a method call `parent()`
But it's fine, I am going to fix this myself.